sklearn/__init__.py

```diff
@@ -16,10 +16,12 @@
 import re
 import warnings
 
+from .exceptions import SklearnDeprecationWarning
 
 # Make sure that DeprecationWarning within this package always gets printed
 warnings.filterwarnings('always', category=DeprecationWarning,
```

Comment:
I kept it for now because it is easy for contributor to miss that DeprecationWarning is no longer the way to go; it may invalidate some of the pull requests.